packages/components/src/ellipsis-menu/test/__snapshots__/index.js.snap

```diff
@@ -0,0 +1,48 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Whats your thought on snapshot tests? I don't see them as being high value, but don't mind keeping them if you see the value.